asreview/datasets.py

```diff
@@ -413,5 +413,23 @@ def __init__(self):
         )
 
 
+class BenchmarkDataGroup(BaseDataGroup):
+    group_id = "benchmark"
+    description = "A collections of fully labeled datasets for benchmarking."
```

Comment:
Rephrase: "A collection of labelled datasets for benchmark testing" 

Do the data need to be fully labelled? The simulation also runs for partly labelled data and the unlabelled data are then just ignored. So, is the "fully labelled" needed for the new benchmark integration?

Reply:
Good question. It depends on what you put in here: https://github.com/asreview/systematic-review-datasets. At the moment, we do have fully labeled datasets only. We should have a good reason when mixing partly and fully labeled datasets over there (however can be a good idea). We need a metadata standard for those datasets and make this part of that metadata. The current metadata version is only a draft: https://github.com/asreview/systematic-review-datasets/blob/master/datasets/van_de_Schoot_2017/van_de_Schoot_2017.json. I know that @bmkramer and Otto do have some ideas/experiences on this.

Reply:
Ok, thanks for explaining! So, in the text we can drop the word 'fully' because it is not a technical requirement. 

I am working on some partly labelled datasets we can add in a later stage.

Reply:
Oké. I will update this. Please keep in mind that for Exploration mode this is a requirement. So, please remember this when adding the partly labeled datasets. That will require a implemented heuristic that's not implemented yet.

Reply:
Ok! Is there a test for the exploration mode to test if the column with labels contains no missing values, and if so, it should provide an error message

Reply:
This is to be implemented. Let's forward this check to the next release (the release that implements the new exploration mode and simulation mode).